tapdb/sqlc/schemas/generated_schema.sql

```diff
@@ -632,6 +619,10 @@ CREATE TABLE proof_transfer_log (
     time_unix TIMESTAMP NOT NULL
 );
 
+CREATE TABLE proof_types (
```

Comment:
Nice part about this is that you can spot check the _end_ result of a migration.

Reply:
yes! very valuable